everviz/plugins/_lineplot_plugin/_dash_components/_lineplot_callback.py

```diff
@@ -0,0 +1,11 @@
+def _lineplot_callback(data, yaxis_column_name):
+    return {
+        "data": data,
+        "layout": dict(
+            yaxis={
+                "title": yaxis_column_name,
+            },
+            xaxis={"title": "batch"},
```

Comment:
Is the `xaxis` title always going to be `batch`?

Reply:
Will address this comment in another PR.